canonical/vanilla-framework
# Add basic themes documentation

## Done

Updates some legacy themes documentation.

Fixes https://warthogs.atlassian.net/browse/WD-7593

## QA

- Review updated documentation:
  - Check new Themes page: https://vanilla-framework-5378.demos.haus/docs/base/themes
  - Check updated colour settings (with deprecated theming section): https://vanilla-framework-5378.demos.haus/docs/settings/color-settings#color-theming
  - Make sure that old paper background page redirects to new themes page: https://vanilla-framework-5378.demos.haus/docs/base/paper

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix release (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical/vanilla-framework/wiki/Release-process#pre-release-tasks)


## Screenshots

<img width="1504" alt="image" src="https://github.com/user-attachments/assets/6031eaf8-2450-45b4-a43f-cedf03e75f2f">
